lib/octokit/error.rb

```diff
@@ -68,6 +68,8 @@ def self.error_for_401(headers)
     def self.error_for_403(body)
       if body =~ /rate limit exceeded/i
```

Comment:
An alternative would be to instead just do:

```suggestion
      if body =~ /rate limit/i
```

However, went with the more explicit approach for now.